nix/commands.nix

```diff
@@ -68,9 +63,32 @@
     docker compose up --build --wait ''${backends[@]}
   '';
 
+  letsql-newgrp-docker-compose-up = pkgs.writeShellScriptBin "letsql-newgrp-docker-compose-up" ''
+    set -eux
+
+    newgrp docker <<<"${letsql-docker-compose-up}/bin/letsql-docker-compose-up ''${@}"
+  '';
+
+  letsql-git-fetch-origin-pull = pkgs.writeShellScriptBin "letsql-git-fetch-origin-pull" ''
+    set -eux
+
+    PR=$1
+    branchname="origin-pr-$PR"
+    git fetch origin pull/$PR/head:$branchname
+  '';
+
+  letsql-git-config-blame-ignore-revs = pkgs.writeShellScriptBin "letsql-git-config-blame-ignore-revs" ''
+    set -eux
+
+    # https://black.readthedocs.io/en/stable/guides/introducing_black_to_your_project.html#avoiding-ruining-git-blame
+    ignore_revs_file=''${1:-.git-blame-ignore-revs}
+    ${pkgs.git}/bin/git config blame.ignoreRevsFile "$ignore_revs_file"
+  '';
+
   letsql-commands = {
-    inherit letsql-pytest letsql-clean letsql-fmt letsql-lint letsql-ensure-download-data letsql-docker-compose-up;
+    inherit letsql-pytest letsql-fmt letsql-lint letsql-ensure-download-data letsql-docker-compose-up letsql-newgrp-docker-compose-up letsql-git-fetch-origin-pull letsql-git-config-blame-ignore-revs;
```

Comment:
One invokes `newgrp` which is required until a re-login after a user is initially added to the `docker` group.
A reason why this could take a while is that an existing `tmux` will prevent other, even new, `tmux` sessions from having the user in the `docker` group.